src/main/java/frc/robot/commands/ArmPID.java

```diff
@@ -43,18 +34,10 @@ public void initialize() {
       kI = 0;
       kD = 0;
 
-      // ks = ArmConstants.kS;
-      // kg = ArmConstants.kG;
-      // kv = ArmConstants.kV;
-      // ka = ArmConstants.kA;
-
-      armVelocity = 500;
-      armAcceleration = 200;
+      pid = new ProfiledPIDController(kP, kI, kD, new TrapezoidProfile.Constraints(ArmConstants.ARM_VELOCITY, ArmConstants.ARM_ACCELERATION));
```

Comment:
What units are your velocity and acceleration in?

Reply:
meters/sec and meters/sec^2 respectively.